src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -39,19 +41,34 @@ class MovingWindow:
     the point in time that defines the alignment can be outside of the time window.
     Modulo arithmetic is used to move the `window_alignment` timestamp into the
     latest window.
-    If for example the `window_alignment` parameter is set to `datetime(1, 1, 1)`
-    and the window size is bigger than one day then the first element will always
-    be aligned to the midnight. For further information see also the
+    If for example the `window_alignment` parameter is set to
+    `datetime(1, 1, 1, tzinfo=timezone.utc)` and the window size is bigger than
+    one day then the first element will always be aligned to the midnight.
+    For further information see also the
     [`OrderedRingBuffer`][frequenz.sdk.timeseries._ringbuffer.OrderedRingBuffer]
     documentation.
 
+    Resampling might be required when working with streams of samples from
+    different sources at different sampling rates or periods, and it can be
+    set by specifying the resampler config parameter so that the user can
+    control the granularity of the samples to be stored in the underlying
+    buffer.
+
+    If resampling is not required, the resampler config parameter can be
+    set to None or the resampling period can be set equal to the input
+    sampling period in which case the MovingWindow will not perform any
```

Comment:
Also I might need to amend this paragraph as `or the resampling period can be set equal to the input sampling period` might no longer apply